src/drivers/glfw/lv_glfw_window.c

```diff
@@ -300,7 +300,10 @@ static void window_update_handler(lv_timer_t * t)
                 lv_refr_now(texture_disp);
             }
 
-            lv_opengles_render_texture(texture->texture_id, &texture->area, texture->opa, window->hor_res, window->ver_res);
+            lv_area_t clip_area = texture->area;
+            lv_area_move(&clip_area, -texture->area.x1, -texture->area.y1);
+            lv_opengles_render_texture(texture->texture_id, &texture->area, texture->opa, window->hor_res, window->ver_res,
+                                       &clip_area, false);
```

Comment:
Did you set the color depth to 32?

Reply:
Thanks for pointing that one out! fixed now

Reply:
> When setting LV_USE_DRAW_OPENGLES to 0, so not using the draw_opengles src, this will now flip the window. Setting the flip field to true resolves this. This either needs an #ifdef LV_USE_DRAW_OPENGLES or some other solution in the opengles.c file.

Good catch! I will fix it.